src/docs/develop/variables.md

```diff
@@ -33,13 +45,83 @@ alt="Screenshot of Raw Editor"
 layout="responsive"
 width={1954} height={1303} quality={100} />
 
-Variables can be defined as simple key/value pairs or as [Templated Variables](#templated-variables) (eg. `${{Postgres.DATABASE_URL}}`),
-which can dynamically reference other variables, shared variables, or plugin variables (more on this below).
+
+#### Using Reference Variables
+
+Reference variables refer to variables set on a service that reference variables which are defined elsewhere, i.e. in another service or Shared variables.
+
+More on reference variables [below](/develop/variables#reference-variables).
```

Comment:
This section is introduced, then buried, then re-introduced later

Propose moving references up so that, as users evolve their application, they can go "deeper" into the docs

Reply:
moving things around made this flow perfectly